<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>122</width>
      <height>167</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>流控:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>DTR/DSR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>RTS/CTS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>XON/XOFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>79</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>端口:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>128</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>波特率:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portcbox">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>79</y>
      <width>220</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>177</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>数据位:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portcbox_2">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>128</y>
      <width>220</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>275</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>停止位:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>226</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>奇偶校验:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portcbox_3">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>177</y>
      <width>220</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="portcbox_4">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>226</y>
      <width>220</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="portcbox_5">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>275</y>
      <width>220</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>31</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
